### PR TITLE
SCMOD-13614: Updated release-notes

### PR DESCRIPTION
**Pull request by xbreizh**
https://portal.digitalsafe.net/browse/SCMOD-13614

### CODE REVIEW

File: release-notes-5.0.0.md [hunk 1]
```diff
@@ -1,22 +1,24 @@
...
 #### Version Number
 ${version-number}
 
 #### New Features
-Correlation Id support has been added to worker-framework which can be used for tracking messages.  
+- SCMOD-12257: Correlation Id support has been added to worker-framework which can be used for tracking messages.  
 The Correlation Id will be included in the output message so downstream workers must be on a version that supports it.
 
+#### Bug Fixes
+- SCMOD-13301: Worker-Framework: Move away from Files.exists()
```
Comment (michael-bryson) [inline release-notes-5.0.0.md]: Re-word this, why would we do this?

File: release-notes-5.0.0.md [hunk 2]
```diff
@@ -1,22 +1,24 @@
...
+- SCMOD-12319: Removed worker archetype
```
Comment (michael-bryson) [inline release-notes-5.0.0.md]:
```suggestion
- SCMOD-12319: The worker archetype has been deprecated.
```

**Review by michael-bryson — CHANGES_REQUESTED** [review]

File: release-notes-5.0.0.md (hunk 1 above)
Comment (xbreizh) [reply release-notes-5.0.0.md]: Done

File: release-notes-5.0.0.md (hunk 2 above)
Comment (xbreizh) [reply release-notes-5.0.0.md]: Done

**Review by michael-bryson — APPROVED** [review]

**Pull request merged**